### PR TITLE
Small fixes to allow compilation in Visual Studio 2015, and get rid of some warnings

### PR DESCRIPTION
**Pull request by sherm1**
I tried building FCL on Windows 10 with Visual Studio 2015 Win64 and got a couple of compilation failures (there was a missing include and use of non-standard constant `M_PI_2`). This PR fixes those and allows FCL to build. However 4 or 13 tests failed -- those need to be addressed but I'm not attempting that in this PR.

There were also a few warnings when I built in Ubuntu 14.04; those are fixed here too. All tests passed there.

### CODE REVIEW

File: test/test_fcl_capsule_capsule.cpp [hunk 1]
```diff
@@ -126,6 +125,7 @@ BOOST_AUTO_TEST_CASE(distance_capsulecapsule_transformZ)
 
 BOOST_AUTO_TEST_CASE(distance_capsulecapsule_transformZ2)
 {
+  const FCL_REAL Pi = std::acos(FCL_REAL(-1.0));
```
Comment (isucan) [inline test/test_fcl_capsule_capsule.cpp]: I would prefer either using boost::math::constants::pi<double> or defining M_PI if not defined.
{{{
#ifndef M_PI
#define M_PI 3.1415926535....
#endif
}}}

Comment (sherm1) [inline test/test_fcl_capsule_capsule.cpp]: I was trying to fix what appeared to be incorrect use of floating point type. Using macros for constants means they will be untyped and might not convert properly to FCL_REAL. I took a look at boost::pi and can't figure out what type it is! I don't think it is a good idea to have the digits of pi included here -- there must be a better way! What would you suggest as the proper way to get a "pi" of type FCL_REAL?

Comment (sherm1) [inline test/test_fcl_capsule_capsule.cpp]: Oh, I see -- it's a template function. I'll use it.

**Comment by sherm1** [comment]
@isucan -- I replaced the calculated pi with the boost version.

**Comment by isucan** [comment]
Thanks!

**Pull request merged**